internal/repository/account_repo_psql.go

```diff
@@ -0,0 +1,95 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/gopay/internal/models"
+)
+
+const (
+	createQ = `
+	INSERT INTO accounts 
+	(name, last_name) 
+	VALUES ($1, $2) 
+	RETURNING account_id
+	`
+
+	findAllQ = `
+	SELECT * 
+	FROM accounts
+	`
+
+	findOneQ = `
+	SELECT *
```

Comment:
Instead of passing `*`, you should specify all the fields that you want.
It is important, especially if you need them in a specific order.

Reply:
This is not specifying the field uet, it is using *